enrolments/models.py

```diff
@@ -33,6 +34,11 @@ class Class(models.Model):
         blank=True,
     )
 
+    attendance = models.JSONField(validators=[validate_attendance])
+
+    class Meta:
+        verbose_name_plural = "classes"
```

Comment:
What is `verbose_name_plural` used for?

Reply:
It tells Django that the plural of "class" is "classes" instead of "classs". Django by default just adds an s to the end.